packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -55,8 +55,8 @@ export class InsecureUrlResolver implements IUrlResolver {
         // If hosts match then we use the local tenant information. Otherwise we make a REST call out to the hosting
         // service using our bearer token.
         if (this.isForNodeTest) {
-            const documentId = parsedUrl.pathname.substr(1).split("/")[1];
-            return this.resolveHelper(documentId, "");
+            const [, documentId, relativePath] = parsedUrl.pathname.substr(1).split("/");
+            return this.resolveHelper(documentId, `/${relativePath}`, parsedUrl.search);
```

Comment:
You can relative path without "/" here and then in resolve helper make sure  if it does not have slash then prefix "/". So in that way user can pass anything for relative path with or without slash.